pkgs/applications/misc/plover/default.nix

```diff
@@ -1,37 +1,138 @@
+            inherit (lib) licenses platforms maintainers;
+          in
+          {
+            description = "Stroke handling helper library for Plover";
+            license = licenses.gpl2Plus;
+            homepage = "https://github.com/openstenoproject/plover_stroke";
+            platforms = platforms.linux ++ platforms.windows;
+            maintainers = [ maintainers.FirelightFlagboy ];
+          };
+      };
+
+      rtf_tokenize = mkDerivationWith buildPythonPackage rec {
+        pname = "rtf_tokenize";
+        version = "1.0.0";
+
+        pyproject = true;
+
+        src = fetchFromGitHub {
+          owner = "openstenoproject";
+          repo = "rtf_tokenize";
+          rev = version;
+          sha256 = "sha256-zwD2sRYTY1Kmm/Ag2hps9VRdUyQoi4zKtDPR+F52t9A=";
+        };
 
-    # I'm not sure why we don't find PyQt5 here but there's a similar
-    # sed on many of the platforms Plover builds for
-    postPatch = "sed -i /PyQt5/d setup.cfg";
+        propagatedBuildInputs = [ setuptools ];
 
-    nativeCheckInputs           = [ pytest mock ];
-    propagatedBuildInputs = [ babel pyqt5 xlib pyserial appdirs wcwidth setuptools ];
+        nativeCheckInputs = [ pytestCheckHook ];
+        pythonImportsCheck = [ "rtf_tokenize" ];
 
-    dontWrapQtApps = true;
+        meta =
+          let
+            inherit (lib) licenses platforms maintainers;
+          in
+          {
+            description = "Simple RTF tokenizer";
+            license = licenses.gpl2Plus;
+            homepage = "https://github.com/openstenoproject/rtf_tokenize";
+            platforms = platforms.linux ++ platforms.windows;
+            maintainers = [ maintainers.FirelightFlagboy ];
+          };
+      };
+    in
+    mkDerivationWith buildPythonPackage rec {
+      pname = "plover";
+      version = "4.0.0rc2";
 
-    preFixup = ''
-      makeWrapperArgs+=("''${qtWrapperArgs[@]}")
-    '';
-  };
+      pyproject = true;
+
+      src = fetchFromGitHub {
+        owner = "openstenoproject";
+        repo = "plover";
+        rev = "v${version}";
+        sha256 = "sha256-rmMec/BbvOJ92u8Tmp3Kv2YezzJxB/L8UrDntTDSKj4=";
+      };
+
+      propagatedBuildInputs = [
+        babel
+        pyqt5
+        xlib
+        pyserial
+        appdirs
+        wcwidth
+        setuptools
+        plover_stroke
+        rtf_tokenize
+      ];
+
+      buildInputs = [ qt5.qtwayland ];
+
+      nativeCheckInputs = [
+        pytestCheckHook
+        mock
+      ];
+
+      dontWrapQtApps = true;
+
+      preFixup = ''
+        makeWrapperArgs+=("''${qtWrapperArgs[@]}")
+      '';
+
+      meta =
+        let
+          inherit (lib) licenses maintainers stdenv;
+        in
+        {
+          mainProgram = "plover";
+          broken = stdenv.hostPlatform.isDarwin;
+          description = "OpenSteno Plover stenography software";
+          maintainers = builtins.attrValue {
+            inherit (maintainers) FirelightFlagboy twey kovirobi;
+          };
```

Comment:
```suggestion
          maintainers = with maintainers; [
          	FirelightFlagboy
          	twey
          	kovirobi
         ];
```

Reply:
I thought it was recommended to use `inherit` over `with` whenever possible.
What's our point of view on that ?